nss-day-cohort-39/nutshell-veridian-vicuna
# Chat Message Functionality

### Changes
- Send button now saves the value of new message input and renders on page.
- Send button requires input to function.
- New message should be displayed according to the current user (sender). 
- Chat list should scroll when many messages are sent.

### Testing
- Click send button before entering chat message and receive an error (alert).
- Click send button after entering chat message and see it appear on the list.
- Chat list should scroll when many messages are sent.
